php/api.php

```diff
@@ -5,12 +5,6 @@
 # @copyright	2015 by Tobias Reich
 ###
 
-@ini_set('max_execution_time', '200');
```

Comment:
I would normally agree, but there're a lot of users not familiar with those settings or PHP settings in general. They often expect things to "just work". Most default values are too low for Lychee.

But … I'm not sure on how many systems `ini_set` is allowed. Maybe it's ok to remove it, because it isn't working for most of the users anyway.